runtime/polkadot/src/lib.rs

```diff
@@ -1118,13 +1047,11 @@ construct_runtime! {
 		TechnicalMembership: membership::<Instance1>::{Module, Call, Storage, Event<T>, Config<T>},
 		Treasury: treasury::{Module, Call, Storage, Event<T>},
 
-		// Parachains stuff; slots are disabled (no auctions initially). The rest are safe as they
-		// have no public dispatchables. Disabled `Call` on all of them, but this should be
-		// uncommented once we're ready to start parachains.
-		Parachains: parachains::{Module, Call, Storage, Config, Inherent, Origin},
```

Comment:
This changes the outer origin enum, this requires an upgrade of scheduler::Agenda storage (or a dummy variant in origin, maybe using Dummy)

(scheduler agenda holds the origin since https://github.com/paritytech/substrate/pull/6387 )

Reply:
Great point. We'll add the parachains origin to this PR

Reply:
We did some digging, and found that the migration is not really necessary because:
1. The council on Polkadot and Kusama has not scheduled anything with a Parachains origin and will not, either
2. The Origin enum had the Parachains origin at the end